types/focil/inclusion_list.yaml

```diff
@@ -0,0 +1,24 @@
+Focil:
+  InclusionList:
```

Comment:
we'll want a fork name rather than a EIP name for the api, but it's probably premature to know that I guess...

Reply:
I think for now we should keep it as EIP name (or number) but it would be good if we can merge new endpoints before those are officially part of a hard fork. I added a new tag "Draft" here to separate those in the api explorer, but maybe even a EIP specific tag would make sense.

Reply:
In teku i just use `Experimental` tag but draft also works, that'd be fine...

Reply:
That's what I [was using](https://github.com/ethereum/beacon-APIs/pull/490/commits/b82eaf152555e4717f8e721adb9c372ae296da18) but it had a different purpose previously when it was used [for the rewards apis](https://github.com/ethereum/beacon-APIs/pull/426/files#diff-962ea34dc6ec0e65d4e1d6c8682e97c547e6e0d2f1168925210ff7ee47ef6ce0L62). I think either one works, if we do this for more EIPs it might even be good to use a separate tag for each. This would allow to disable spec tests based on tag

Reply:
its been super problematic merging with feature flags sometimes, that's my hesitation.

Reply:
Yeah we probably won't be merging this any time soon since the CL spec isn't even merged yet but it would be nice if we can come up with an approach to allow merging this before it's confirmed as part of a hard fork.

I pushed a few changes group files / types under EIP specific namespace and used the EIP as tag as well, this would allow us to have more than one EIP in "Draft" stage and be part of the api explorer.